### PR TITLE
Add nano for setting up VS Code for Vue

### PR DESCRIPTION
**Pull request by lauragift21**
Nano for Setting up VS Code for Vue.

### CODE REVIEW

File: workshop/nanos/nano3.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+
+# We're going to learn how to setup VScode the right way for Vue
+
+In this activity we are going to walkthrough how to configure your VS Code workspace for appropiate development with Vue. As a developer working with Vue, it is a great deal to msetup your work tool to increase productivity whereby you use tool that already do most of the work you neccessarily don't need to bother about.
+
+## Get Started
+
+In order to start developing Vue applications. A required step is to first setup your workspace or code editor for development and this nano is going to highlight how to setup VS Code for Vue.
+
```
Comment (jlooper) [inline workshop/nanos/nano3.md:17]: You might want to add a bit here outlining the elements that are necessary specifically for a Vue developer. Why, Vetur, for example? What are the pain points specific to Vue?

File: workshop/nanos/nano3.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Automatic formatting can either be done on save command or there's an option to format entire document.
+
+![format](./images/format.png)
+
+## IntelliSense
+Vetur offers IntelliSense all over your .vue file by default.
+
+![IntelliSense](./images/intelliSense.png)
+
```
Comment (jlooper) [inline workshop/nanos/nano3.md:102]: Are there other things other than Vetur that need to be installed? Wonder what Sarah Drasner's pack for the Vue dev offers. Could we also add a plugin for Git as well?

File: workshop/nanos/nano3.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+## Conclusion and challenge
+This is a great way of configuring your work environment for development with Vue. Try going through every step outlined in this tutorial and setup your own VS Code environment,testing out all the features outlined in this nano.
+
```
Comment (jlooper) [inline workshop/nanos/nano3.md:105]: I think we don't need a challenge here; we can remove that bit  and just keep the Conclusion

**Review by jlooper — COMMENTED** [review]
I've added some comments on how to expand this a bit....passing over to @NataliaTepluhina

File: workshop/nanos/nano3.md (hunk 2 above)
Comment (lauragift21) [reply workshop/nanos/nano3.md:102]: Cool. I could add more details about other extensions inclusive of Git lens and Prettier.

File: workshop/nanos/nano3.md (hunk 3 above)
Comment (lauragift21) [reply workshop/nanos/nano3.md:105]: Alright noted. Will take it out.

**Comment by jlooper** [comment]
will merge and copyedit! Thank you!

**Pull request merged**